### ISSUE TITLE
gnt-instance remove does not remove directory on shared file storage path

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#318]
Originally reported of Google Code with ID 262.

```
What software version are you running? Please provide the output of "gnt-
cluster --version" and "gnt-cluster version".
# gnt-cluster --version
gnt-cluster (ganeti v2.6.0) 2.6.0
# gnt-cluster version
Software version: 2.6.0
Internode protocol: 2060000
Configuration format: 2060000
OS api version: 20
Export interface: 0

<b>What distribution are you using?</b>
# cat /etc/SuSE-release
SUSE Linux Enterprise Server 11 (x86_64)
VERSION = 11
PATCHLEVEL = 2

<b>What steps will reproduce the problem?</b>
1. gnt-instance add --no-install --no-name-check --no-ip-check -t sharedfile --disk 0:size=10G -n node01 -o someos+default testvm.loc
2. # ls -ald /srv/ganeti/shared-file-storage/testvm.loc/
drwx------ 2 nobody nobody 4096 Aug 13 13:07 /srv/ganeti/shared-file-storage/testvm.loc/
3. gnt-instance remove testvm.loc
4. ls -ald /srv/ganeti/shared-file-storage/testvm.loc/
drwx------ 2 nobody nobody 4096 Aug 13 13:09 /srv/ganeti/shared-file-storage/testvm.loc/

<b>What is the expected output? What do you see instead?</b>
gnt-instance remove should also remove the directory /srv/ganeti/shared-file-storage/testvm.loc/

```
Originally added on 2012-08-13 11:13:11 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2012-10-30 11:28:30 +0000 UTC.

Added Labels: Type-Defect
Changed State: Started
Added to Milestone: Release2.6

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2012-10-30 11:39:15 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2012-10-30 12:19:15 +0000 UTC.

Changed State: Fixed

**Comment by Ganeti-Issues-Migrator** [comment]
```
Marking as released the bugs fixed in 2.6.2.
```
Originally added on 2013-01-11 12:13:05 +0000 UTC.

Changed State: Released
